recipe/meta.yaml

```diff
@@ -32,8 +32,6 @@ outputs:
       files:
         - test.cu
       commands:
-        # Verify the symlink to the libcuda stub library exists.
-        - test -f "$(${CC} --print-sysroot)/lib/libcuda.so"
```

Comment:
Why move this down? This should have been created by activate automatically

Reply:
I had a brainfart and started thinking I needed to wait until after the activate is called in the tests here, will revert.